src/convertToObject.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const finalStyle = {};
+
+  sourceString.split(';').forEach((part) => {
+    if (part.trim()) {
+      const [command, value] = part.split(':');
+
+      finalStyle[command.trim()] = value.trim();
+    }
+  });
+
+  return finalStyle;
```

Comment:
While the function logic is correct, the variable naming 'finalStyle' could be improved to better describe its purpose. According to the checklist, variable names like 'object' or 'result' decrease code readability. Consider renaming 'finalStyle' to something more descriptive, such as 'styleObject' or 'cssProperties', to clearly indicate what the variable represents.